Gem/Code/Source/Components/NetworkAiComponent.cpp

```diff
@@ -25,34 +25,11 @@ namespace MultiplayerSample
     {
     	if (IsNetEntityRoleAutonomous() && mps_botMode)
         {
-            SetEnabled(true);
+            //SetEnabled(true);
         }
 	}
 
-    void NetworkAiComponentController::OnActivate([[maybe_unused]] Multiplayer::EntityIsMigrating entityIsMigrating)
-    {
-        if (GetEnabled())
-        {
-            Multiplayer::LocalPredictionPlayerInputComponentController* playerInputController = GetLocalPredictionPlayerInputComponentController();
-            if (playerInputController != nullptr && !IsNetEntityRoleAutonomous())
-            {
-                playerInputController->ForceEnableAutonomousUpdate();
-            }
-        }
-    }
-
-    void NetworkAiComponentController::OnDeactivate([[maybe_unused]] Multiplayer::EntityIsMigrating entityIsMigrating)
-    {
-        if (GetEnabled())
-        {
-            Multiplayer::LocalPredictionPlayerInputComponentController* playerInputController = GetLocalPredictionPlayerInputComponentController();
-            if (playerInputController != nullptr && !IsNetEntityRoleAutonomous())
-            {
-                playerInputController->ForceDisableAutonomousUpdate();
```

Comment:
Do we still need this logic?

Reply:
I'm not 100% sure yet. I need to dig more into mps_botMode and get that functioning again before I make cuts.

Reply:
I'll be removing this in favor of making NetworkAiComponent a server only concept and adding separate but similar machinery for local bot mode. This'll enable changing NetworkAi to something other than a chaos bot down the line.